napari/utils/color.py

```diff
@@ -15,6 +15,9 @@ class ColorValue(np.ndarray):
     use the ``validate`` method to coerce a value to a single color.
     """
 
+    def __new__(cls, input_array):
+        return cls.validate(input_array)
```

Comment:
See https://github.com/napari/napari/pull/4704#discussion_r900289069. Basically lets us use this as a pure `np.ndarray` while having our own custom constructor and validation.